dsp/dsp.py

```diff
@@ -0,0 +1,254 @@
+import numpy as np
+import scipy.signal as sig
+
+window_types = [
+    'boxcar',
+    'bartlett',
+    'hann',
+    'blackman',
+    'blackmanharris'
+]
+window_sizes = [64, 128, 256, 512, 1024, 2048, 4096]
+
+def compute_cola_window(config):
```

Comment:
Similar to the other file, can we explicitly pass `window_type` and `window_size`?

Reply:
sure, I think that makes sense here. Originally I was adding the return values of this function to the config object, so it made sense to just pass in the params that way. So we could also put it back that way and just lean into the 'incrementally build up the config object' approach you're so fond of ;)